src/App/Components/DisplayArticle/Ratings.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable react/no-unused-state */
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { ToastContainer, toast } from 'react-toastify';
+import propTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import Rater from 'react-star-ratings';
+import Rating from './Rating';
+import actions from '../../../Redux/Actions/articleRatings';
+
+const { getRatings, rate } = actions;
+
+export class Ratings extends Component {
+  static propTypes = {
+    getRatings: propTypes.func,
+    ratings: propTypes.object,
+    errorMessage: propTypes.string,
+    successMessage: propTypes.string,
+    rate: propTypes.func
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      average: undefined,
```

Comment:
@nkpremices any reason why you're using undefined as opposed to null?

check this out: https://medium.com/@arieldi/anyway-stop-setting-your-variables-to-undefined-difference-undefined-vs-null-25f15d907fb4

Reply:
Thank you @MemunaHaruna 
Let me put it to null according to what I've learned from the ressource

Reply:
@nkpremices did you forget to change the undefined to null here?

Reply:
Oh!! I forgot @MemunaHaruna 
Sorry.
Lemme do that